c++/src/kj/time.c++

```diff
@@ -86,7 +86,7 @@ public:
   }
 
 private:
-  static GetSystemTimePreciseAsFileTimeFunc* getGetSystemTimePreciseAsFileTime() {
+  static GetSystemTimePreciseAsFileTimeFunc* const getGetSystemTimePreciseAsFileTime() {
```

Comment:
I don't understand. According to the diff, there wasn't any `const` qualifier present previously.

Did you maybe intend to modify line 75 instead?

Reply:
I did. My bad. I copied the change over by hand. Let me update this.